File: be/src/main/java/com/netcracker/edu/backend/entity/Film.java
package com.netcracker.edu.backend.entity;

import javax.persistence.*;

@Entity
@Table(name="film")
public class Film {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id_FILM;
    private String name;
    private Long date_release;
    private String description;
    private Double cost_rent;
    private Double cost_buy;

    public Film() {
    }

    public Film(String name, Long date_release, String description, Double cost_rent, Double cost_buy) {
        this.name = name;
        this.date_release = date_release;
        this.description = description;
        this.cost_rent = cost_rent;
        this.cost_buy = cost_buy;
    }

    public Long getId_FILM() {
        return id_FILM;
    }

    public void setId_FILM(Long id_FILM) {
        this.id_FILM = id_FILM;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getDate_release() {
        return date_release;
    }

    public void setDate_release(Long date_release) {
        this.date_release = date_release;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Double getCost_rent() {
        return cost_rent;
    }

    public void setCost_rent(Double cost_rent) {
        this.cost_rent = cost_rent;
    }

    public Double getCost_buy() {
        return cost_buy;
    }

    public void setCost_buy(Double cost_buy) {
        this.cost_buy = cost_buy;
    }
}

File: frontend/src/app/model/film.ts
/**
 * Created by Alexey on 13.11.2018.
 */

export class Film {
  id_FILM: string;
  name: string;
  date_release: string;
  description: string;
  cost_rent: string;
  cost_buy: string;

  static cloneBase(film: Film): Film {
    let clonedFilm: Film = new Film();
    clonedFilm.id_FILM = film.id_FILM;
    clonedFilm.name = film.name;
    clonedFilm.date_release = film.date_release;
    clonedFilm.description = film.description;
    clonedFilm.cost_rent = film.cost_rent;
    clonedFilm.cost_buy = film.cost_buy;
    return clonedFilm;
  }
}
